Gems/AtomLyIntegration/CommonFeatures/Assets/Editor/Scripts/LegacyContentConversion/LegacyConversionHelpers.py

```diff
@@ -101,30 +101,36 @@ class Common_Command_Line_Options(object):
     """
     Some common options/parsing
     """
-    def __init__(self, argv0, argv1):
-        arguments = argv1.split('-')
-
+    def __init__(self, args):
         self.projectName = ""
+        self.assetCatalogOverridePath = ""
         self.includeGems = False
         self.useP4 = False
         self.endsWithStr = ""
         self.isHelp = False        
-        self.helpString = "usage: {0} -project=<project name> -include_gems -ends_with=<filter> -use_p4\n\
+        self.helpString = f"usage: {args[0]} -project=<project name> -include_gems -ends_with=<filter> -use_p4 -assetCatalogOverridePath=<path>\n\
```

Comment:
Not really a comment on these changes, but the original argument parsing would've been easier if pythons [argparse](https://docs.python.org/3.10/library/argparse.html) was used here.

Reply:
Yeah I agree :) However I prefer to not do the refactor via this PR